runner/operator/access/v1_0_0/snps_and_indels/__init__.py

```diff
@@ -0,0 +1,174 @@
+"""""""""""""""""""""""""""""
+
+import json
+from jinja2 import Template
+
+from runner.models import Port
+from runner.operator.operator import Operator
+from runner.serializers import APIRunCreateSerializer
+from file_system.repository.file_repository import FileRepository
+
+
+def get_curated_normals(patient_id):
+    """
+    Return curated normal bams as yaml file objects
+
+    :return: (list, list)
+    """
+    normal_bams = FileRepository.filter(
+        metadata={
+            'patientId': patient_id,
+            'tumorOrNormal': 'Normal',
+            'igocomplete': True
+        }
+    )
+    normal_bams = [{'class': 'File', 'location': b.path} for b in normal_bams]
+    normal_ids = [n['metadata']['cmoSampleId'] for n in normal_bams]
+    return normal_bams, normal_ids
+
+
+def construct_sample_inputs(patient_id, tumor_bam, tumor_simplex_bam, tumor_sample_id, matched_normal_bam, normal_sample_id):
+    with open('runner/operator/access/v1_0_0/snps_and_indels/input_template.json.jinja2') as file:
+        template = Template(file.read())
+
+        tumor_sample_names = [tumor_sample_id]
+        tumor_bams = [{
+            "class": "File",
+            "location": "juno://" + tumor_bam['path']
+        }]
+
+        normal_sample_names = ['']
+        matched_normal_ids = [normal_sample_id]
+
+        # Todo: how to know which sequencer's default normal to use?
+        normal_bams = [{
+            "class": "File",
+            "location": "/ifs/work/bergerm1/ACCESS-Projects/hiseq_curated_duplex/F22_cl_aln_srt_MD_IR_FX_BR__aln_srt_IR_FX-duplex.bam"
+        }]
+
+        genotyping_bams = [
+            {
+                "class": "File",
+                "location": tumor_bam['metadata']['path']
+            },
+            {
+                "class": "File",
+                "location": tumor_simplex_bam['metadata']['path']
+            },
+            {
+                "class": "File",
+                "location": matched_normal_bam['metadata']['path']
+            }
+        ]
+        genotyping_bams_ids = [tumor_sample_id, tumor_sample_id + '-SIMPLEX', normal_sample_id]
+
+        curated_normal_bams, curated_normal_ids = get_curated_normals(patient_id=patient_id)
+        genotyping_bams += curated_normal_bams
+        genotyping_bams_ids += curated_normal_ids
+
+        input_file = template.render(
+            tumor_bams=json.dumps(tumor_bams),
+            normal_bams=json.dumps(normal_bams),
+            tumor_sample_names=json.dumps(tumor_sample_names),
+            normal_sample_names=json.dumps(normal_sample_names),
+            matched_normal_ids=json.dumps(matched_normal_ids),
+            genotyping_bams=json.dumps(genotyping_bams),
+            genotyping_bams_ids=json.dumps(genotyping_bams_ids),
+        )
+
+        sample_input = json.loads(input_file)
+        return sample_input
+
+
+class AccessLegacySNVOperator(Operator):
+    def get_jobs(self):
+
+        run_ids = self.run_ids
```

Comment:
Instead of using the run_id, filter FileRepo for all tumor bams with no associated run through variant calling

That way we can run these bams from the previous pipeline, but also ones that may have missed being run due to previously having no paired normal